From ea0ef98604074fccab6e1ffdbe0ea5d05a91ee45 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@nxp.com>
Date: Wed, 22 Jun 2016 10:48:41 -0500
Subject: [PATCH 1103/5242] MLK-12944 fix makefile miss
 imx7d-12x12-lpddr3-arm2-pcie.dtb

commit  bd7faa11c8a26ce8696ac7ad2fe66da194d1927c from
https://source.codeaurora.org/external/imx/linux-imx.git

Added imx7d-12x12-lpddr3-arm2-pcie.dtb to makefile

Signed-off-by: Frank Li <Frank.Li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 6c89e67..a284b94 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -653,6 +653,7 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-12x12-lpddr3-arm2-qspi.dtb \
 	imx7d-12x12-lpddr3-arm2-sai.dtb \
 	imx7d-12x12-lpddr3-arm2-mqs.dtb \
+	imx7d-12x12-lpddr3-arm2-pcie.dtb \
 	imx7d-19x19-lpddr2-arm2.dtb
 dtb-$(CONFIG_SOC_LS1021A) += \
 	ls1021a-moxa-uc-8410a.dtb \
-- 
1.7.9.5

